### PR TITLE
Correction Suggestions LL: Event Sourcing Is Not My Choice

### PR DESCRIPTION
**Pull request by LouisLouw1**

### CODE REVIEW

File: _posts/2017-03-04-event-sourcing-is-not-a-choice.md [hunk 1]
```diff
@@ -68,69 +68,71 @@ this is the possible set:
...
-you can think about ES.
+Well, then perhaps you can think about ES.
 
 The best examples of applying event-driven architecture are 
 [Blockchain](https://en.wikipedia.org/wiki/Blockchain_(database)) and 
-[Bitcoin](https://en.wikipedia.org/wiki/Bitcoin). Of course, everything
-related to storing transactions is potential candidate. But not everything
-that moves.
+[Bitcoin](https://en.wikipedia.org/wiki/Bitcoin). Essentially, anything
+related to storing transactions could come to mind to here. However, as my blog post has shown,
```
Comment (Shpota) [inline _posts/2017-03-04-event-sourcing-is-not-a-choice.md]: `could come to mind to` typo?

File: _posts/2017-03-04-event-sourcing-is-not-a-choice.md [hunk 2]
```diff
@@ -68,69 +68,71 @@ this is the possible set:
...
+Well, then perhaps you can think about ES.
 
 The best examples of applying event-driven architecture are 
 [Blockchain](https://en.wikipedia.org/wiki/Blockchain_(database)) and 
-[Bitcoin](https://en.wikipedia.org/wiki/Bitcoin). Of course, everything
-related to storing transactions is potential candidate. But not everything
-that moves.
+[Bitcoin](https://en.wikipedia.org/wiki/Bitcoin). Essentially, anything
+related to storing transactions could come to mind to here. However, as my blog post has shown, 
+ES becomes problematic when dealing with anything "on-the-move".
```
Comment (Shpota) [inline _posts/2017-03-04-event-sourcing-is-not-a-choice.md]: The old phrase, `But not everything that moves.`, was a bad attempt to translate Ukrainian idiom to English. I only wanted to say that event-driven architecture can't be applied everywhere. 

Can we change the last phrase to `But make sure not to apply event-driven architecture when it is not needed.`?

File: _posts/2017-03-04-event-sourcing-is-not-a-choice.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 comments: true
 ---
-Event-driven architecture becomes very popular nowadays. It is
-easy to start and it can be easily integrated with popular Java frameworks 
-like Spring. I have been working with event sourcing for the last half a year and 
-I collected some thoughts on this subject. Despite all its features, I believe 
-that such architecture should be applied very carefully with understanding 
-of all of its consequences. 
+Event-driven architecture has become extemely popular nowadays. It is
+easy to start and it can easily be integrated into/with popular Java frameworks
```
[the remark this answers is not in the document]
Comment (LouisLouw1) [reply _posts/2017-03-04-event-sourcing-is-not-a-choice.md]: "into" deleted on commit, "with" included

File: _posts/2017-03-04-event-sourcing-is-not-a-choice.md (hunk 2 above)
Comment (LouisLouw1) [reply _posts/2017-03-04-event-sourcing-is-not-a-choice.md]: Yeah, I was wondering about that one. I wasn't sure if it was Github jargon so I tried to do something along the lines of your original phrasing with movement... I changed it now to something that's more natural

**Comment by LouisLouw1** [comment]
One thing you might want to check in the Docker article is the Capitalisation of the word "Docker". Not sure if it was capitalised or not. It may be that "docker" and "Docker" both appear. I have something like that in mind.

**Pull request merged**